bindings_ffi/src/mls.rs

```diff
@@ -6967,7 +6967,7 @@ mod tests {
         // Wait for alix_a to send out the consent on the sync group
         alix_a
             .worker()
-            .wait(SyncMetric::ConsentSent, 3)
```

Comment:
I'm suspecting these tests were relying on racey behavior or error retries. After the changes in this PR, the third `ConsentSent` now never arrives. The reason I'm comfortable making these changes:

1. When I reason through the logic of the tests, there should only be two consent records sent.
2. These lines are used to synchronize timing before the final consent state is checked. The final consent state is still correct after this change.